### PR TITLE
rfc 64

### PR DESCRIPTION
**Pull request by FRidh**

### CODE REVIEW

File: rfcs/0064-documentation-format.md [hunk 1]
```diff
@@ -79,13 +82,27 @@ Links:
...
-[Primer](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html), [Demo](https://docutils.readthedocs.io/en/sphinx-docs/user/rst/demo.html)
+Language:
+- [Specification](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html)
+- [Primer](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html)
+- [Demo](https://docutils.readthedocs.io/en/sphinx-docs/user/rst/demo.html)
 
 Tooling:
 - [Sphinx](https://www.sphinx-doc.org/)
 - [Docutils](https://docutils.sourceforge.io/)
+- [Pandoc](https://pandoc.org/) can convert from/to Markdown to/from many other formats
```
Comment (infinisil) [inline rfcs/0064-documentation-format.md]: s/Markdown/reST

Comment (FRidh) [reply rfcs/0064-documentation-format.md]: Done

**Comment by infinisil** [comment]
Thanks! Ready for merge or are you still doing some changes?

**Comment by FRidh** [comment]
I think this can go in. Enough for today :)

**Pull request merged**